### DISCUSSION TITLE
关于加入DroidCast_Raw截图方式

### DISCUSSION THREAD

**Discussion by drunkenQCat** [MaaAssistantArknights/MaaAssistantArknights#11356] [Q&A] [answered]
[相关issue](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/11242#issuecomment-2536480462)
我注意到截图函数`asst::AdbController::screencap`的依赖`m_platform_io->call_command`方法。具体是运行`std::string& cmd`所对应的adb命令，获取管道中返回的信息，完成截图的。
但DroidCast_Raw是通过get请求获取截图，而如果要保持上述依赖结构，就要使用系统内支持的类似curl的命令。但windows下默认没有curl，使用powershell下的`Invoke-RestMethod`速度又太慢。不知老哥们有没有什么解法？

**Comment by ABA2396** [comment, marked as answer]
其实感觉不用考虑这么多，不能用的就保持原来的方式呗。反正会过一遍测速，用不了的人就不用

**Reply by drunkenQCat** [reply]
got it. 那我就先这么凑合了。
